### PR TITLE
add mochawesome report

### PR DESCRIPTION
**Pull request by cardel**
image

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -207,4 +207,7 @@ $RECYCLE.BIN/
 # Windows shortcuts
 *.lnk
 
-# End of https://www.toptal.com/developers/gitignore/api/node,windows,linux,visualstudiocode
\ No newline at end of file
+# End of https://www.toptal.com/developers/gitignore/api/node,windows,linux,visualstudiocode
+
+#Reports
+report
```
Comment (holgiosalos) [inline .gitignore]: EOF

File: package.json [hunk 2]
```diff
@@ -27,6 +27,7 @@
     "mocha": "^10.0.0"
   },
   "dependencies": {
-    "axios": "^0.27.2"
+    "axios": "^0.27.2",
+    "mochawesome": "^7.1.3"
```
Comment (holgiosalos) [inline package.json:29]: Install as dev dependency

File: .gitignore (hunk 1 above)
Comment (cardel) [reply .gitignore]: Fixed in commit add mochawesome reportfix

File: package.json (hunk 2 above)
Comment (cardel) [reply package.json:29]: Fixed in commit add mochawesome reportfix

**Review by holgiosalos — APPROVED** [review]

**Pull request merged**